review-of-international-studies.csl

```diff
@@ -112,7 +120,7 @@
               <date date-parts="year" form="text" variable="issued" prefix="(" suffix=")"/>
             </group>
             <group delimiter=" ">
-              <label plural="always" variable="page" form="short"/>
+              <label plural="always" variable="locator" form="short"/>
```

Comment:
the `plural="always"` here is your problem. That forces pp. Take it out & this will work as intended